ix-dev/test/glances/ix_values.yaml

```
@@ -0,0 +1,9 @@
+images:
+  image:
+    repository: nicolargo/glances
+    tag: "4.3.0.7"
+
+consts:
+  container_name: glances
+  perms_container_name: permissions
+  socket_path: /var/run/docker.sock
```

```suggestion
  glaces_container_name: glances
```

socket path is not used